### PR TITLE
Add blog post about streaming event done in June

### PR DESCRIPTION
**Pull request by flavio**
As per title, this fixes issue https://github.com/kubewarden/kubewarden.io/issues/18

### CODE REVIEW

File: content/blog/2021/07/learn-kubewarden-streaming-event.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+as Open Policy Agent and Kyverno.
+
+The talk features also a brief overview of WebAssembly, what it is and what are
+the benefits it provides to Kubewarden.
+
+Finally, the streaming event ends with a live demo that shows Kubewarden in
+action.
+
+During the demo we also showed some experimental code we have been working on.
+This code open a really exciting opportunity: the ability
```
Comment (ereslibre) [inline content/blog/2021/07/learn-kubewarden-streaming-event.md:23]:
```suggestion
This code opens a really exciting opportunity: the ability
```

**Review by ereslibre — APPROVED** [review]
Minor typo, LGTM otherwise! :clap:

File: content/blog/2021/07/learn-kubewarden-streaming-event.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Let's learn Kubewarden - Streaming Event
+authors:
+    - Flavio Castelli
+date: 2021-07-13
+---
+
+In case you missed, CNCF Ambassador [Saiyam Pathak](https://twitter.com/SaiyamPathak)
+recently hosted a live streaming event on his YouTube channel about Kubewarden.
```
Comment (kkaempf) [inline content/blog/2021/07/learn-kubewarden-streaming-event.md:9]:
```suggestion
recently hosted a live streaming event on [his YouTube channel](https://www.youtube.com/saiyam911) about Kubewarden.
```

File: content/blog/2021/07/learn-kubewarden-streaming-event.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+as Open Policy Agent and Kyverno.
+
+The talk features also a brief overview of WebAssembly, what it is and what are
+the benefits it provides to Kubewarden.
+
+Finally, the streaming event ends with a live demo that shows Kubewarden in
+action.
+
+During the demo we also showed some experimental code we have been working on.
+This code opens a really exciting opportunity: the ability
```
Comment (kkaempf) [inline content/blog/2021/07/learn-kubewarden-streaming-event.md:23]:
```suggestion

This code opens a really exciting opportunity: the ability
```

**Review by kkaempf — APPROVED** [review]

**Pull request merged**